src/ValueObject/GalleryItemValueObject.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme\ValueObject;
+
+/**
+ * Handle information about a gallery item.
+ */
+class GalleryItemValueObject extends ValueObjectBase {
+
+  /**
+   * The icon of the gallery item.
+   *
+   * @var string
+   */
+  const ICON = 'camera';
```

Comment:
After a closer look I've realized that we miss the following factory on this object: `GalleryItemValueObject::fromImageField(...)` this will create a gallery item starting from an image Drupal field and it's there that we need to set the icon to `camera`.

So let's put back the property here and set it only in factory methods, as factory methods will have knowledge of the gallery item type they construct.

Reply:
:+1: